### ISSUE TITLE
Help! BufferedCameraOV7670_QQVGA_10hz.h; No such file or directory

### ISSUE THREAD

**Issue by GijsLB** [indrekluuk/LiveOV7670#20]
I have an error when I verify the whole project. The ExampleTftBufferedCameraFrame.cpp file shows an error as such above. I am quiet new to Arduino, but love to learn. Can someone help? :) thanks in advance

**Comment by indrekluuk** [comment]
copy "src/lib/LiveOV7670Library" and "src/lib/Adafruit_GFX_Library" to Arduino "libraries" folder (If you already have "Adafruit_GFX_Library" then you don't have to copy that)

**Comment by GijsLB** [comment]
Hey! Both those libraries are installed; I did it manually. They also appear in Sketch > Include Library

**Comment by indrekluuk** [comment]
This is strange. Did you restart Arduino IDE after installing the libraries?

Alternatively, you can try to copy all the content of the libraries into the sketch folder (next to the .ino file). You may be required to replace the #include <> with #include ""

**Comment by GijsLB** [comment]
Yes, I restarted the Arduino IDE after installing the libraries. By copying all the library content into the sketch folder ( ctr + k) (right?) there seems to be no difference, also after restarting the file. I must be doing something wrong. See the image I've attatched. The right two folders are the libraries, the left folder is the sketch folder. Many thanks
![Probleem LiveOV7670](https://user-images.githubusercontent.com/73178086/102510920-76ee7100-4088-11eb-8a4d-e8ca1c27448b.jpg)

Greetings from Holland

**Comment by indrekluuk** [comment]
If you copy the content next to the .ino file then you should also change the <> to ""
For example 
#include <BufferedCameraOV7670_QQVGA_10hz.h>
to
#include "BufferedCameraOV7670_QQVGA_10hz.h"

But did you at first copy the two libraries into the "C:\Users\<your user name>\Documents\Arduino\libraries" folder? This is the correct location and it should work. It is strange that it didn't work for you.

**Comment by N8delta** [comment]
I have the same problem

**Comment by nkravitz99** [comment]
I'm having the same issue, has anyone been able to resolve it?
